generator/generator.go

```diff
@@ -47,9 +47,17 @@ func WithFileName(name string) GenerateOpt {
 	}
 }
 
+func WithExtractStrings() GenerateOpt {
+	return func(g *generator) error {
+		g.w.extractStrings = true
+		g.w.strings = []string{""}
```

Comment:
This looks weird to me. Does it really need to have an empty string in it? No big deal, just a head scratcher. 😁

Reply:
Cleaned this up